tests/api/test_dnf_module_package.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import os
+
+import dnf
+import libdnf
+
+from .common import TestCase
+
+
+class DnfModulePackageApiTest(TestCase):
+    def setUp(self):
+        self.base = dnf.Base()
+        repo = self.base.repos.add_new_repo(
+            'api-module-test-repo', self.base.conf,
+            baseurl=[os.path.join(os.path.dirname(__file__), "../modules/modules/_all/x86_64/")]
+        )
+        self.base.fill_sack(load_system_repo=False, load_available_repos=True)
+        moduleBase = dnf.module.module_base.ModuleBase(self.base)
+        modulePackages, nsvcap = moduleBase.get_modules('*')
+        self.modulePackage = modulePackages[0]
+
+    def tearDown(self):
+        self.base.close()
+
+    def test_getName(self):
+        # ModulePackage.getName()
+        self.assertHasAttr(self.modulePackage, "getName")
+        self.modulePackage.getName()
+
+    def test_getStream(self):
+        # ModulePackage.getStream()
+        self.assertHasAttr(self.modulePackage, "getStream")
+        self.modulePackage.getStream()
+
+    def test_getVersion(self):
+        # ModulePackage.getVersion()
+        self.assertHasAttr(self.modulePackage, "getVersion")
+        self.modulePackage.getVersion()
+
+    def test_getVersionNum(self):
+        # ModulePackage.getVersionNum()
+        self.assertHasAttr(self.modulePackage, "getVersionNum")
+        self.modulePackage.getVersionNum()
+
+    def test_getContext(self):
+        # ModulePackage.getContext()
+        self.assertHasAttr(self.modulePackage, "getContext")
+        self.modulePackage.getContext()
+
+    def test_getArch(self):
+        # ModulePackage.getArch()
+        self.assertHasAttr(self.modulePackage, "getArch")
+        self.modulePackage.getArch()
+
+    def test_getNameStream(self):
+        # ModulePackage.getNameStream()
+        self.assertHasAttr(self.modulePackage, "getNameStream")
+        self.modulePackage.getNameStream()
+
+    def test_getNameStreamVersion(self):
+        # ModulePackage.getNameStreamVersion()
+        self.assertHasAttr(self.modulePackage, "getNameStreamVersion")
+        self.modulePackage.getNameStreamVersion()
+
+    def test_getFullIdentifier(self):
+        # ModulePackage.getFullIdentifier()
+        self.assertHasAttr(self.modulePackage, "getFullIdentifier")
+        self.modulePackage.getFullIdentifier()
+
+    def test_getProfiles(self):
+        # ModulePackage.getProfiles()
+        self.assertHasAttr(self.modulePackage, "getProfiles")
+        self.modulePackage.getProfiles()
```

Comment:
``getProfiles`` takes argument ``name``. Please add the argument (can be with value ``None``) to test that the method really accepts it.